### PR TITLE
Removes hack_progress method.

### PR DESCRIPTION
**Pull request by ashercodeorg**
If anything, this belongs as a script in `bin/oneoff`, though I'm pretty sure no one will miss this.

### CODE REVIEW

File: dashboard/app/models/user.rb [hunk 1]
```diff
@@ -965,33 +965,6 @@ def can_pair_with?(other_user)
...
-  # used by any user-facing features.
-  def hack_progress(options = {})
-    options[:script_id] ||= Script.twenty_hour_script.id
-    script = Script.get_from_cache(options[:script_id])
-
-    options[:levels] ||= script.script_levels.count / 2
-
-    script.script_levels[0..options[:levels]].each do |sl|
-      # create some fake testresults
-      test_result = rand(100)
```
Comment (ashercodeorg) [inline dashboard/app/models/user.rb:979]: This is bad, given that not all integers are "valid" result scores.

**Comment by Hamms** [comment]
Do we know how this was previously used?

**Comment by Hamms** [comment]
Yep, I'm definitely glad to see this go. LGTM.

**Comment by ashercodeorg** [comment]
Added in https://github.com/code-dot-org/code-dot-org/commit/d442811a29f9fc2d831404c01c3dc6a49bbe46aa. Not sure how it was used.

**Pull request merged**